docs/faq.rst

```diff
@@ -734,6 +734,12 @@ connections and release the lock).
 The borg cache eats way too much disk space, what can I do?
 -----------------------------------------------------------
 
+This may especially happen if borg needs to recreate the local cache based 
+on repository data, either because local cache was removed, or because of multiple
+machines writing to same repository. Borg uses a smaller cache if the repo is never accessed
+from a different cache, but on rebuild switches to a "format" that is easier
+to update but much bigger.
+
 There is a temporary (but maybe long lived) hack to avoid using lots of disk
 space for chunks.archive.d (see :issue:`235` for details):
```

Comment:
did you read issue 235?

Reply:
Read, yes, but I don't think I was able to understand it properly :(

Reply:
How about:

This may especially happen if borg needs to rebuild the local "chunks" index - either because it was removed, or because it was not coherent with the repository state any more (e.g. because another borg changed the repository).

To optimize this rebuild process, borg caches per-archive information in the "chunks.archive.d/" directory.
It won't be faster the first time that happens, but on subsequent rebuilds (because it needs to transfer less data from the repository). While being faster, the cache needs quite some disk space, which might be unwanted.

Reply:
Mmm, nice. How about slightly reworded?

This may especially happen if borg needs to rebuild the local "chunks" index -
either because it was removed, or because it was not coherent with the
repository state any more (e.g. because another borg instance changed the
repository).

To optimize this rebuild process, borg caches per-archive information in the
``chunks.archive.d/`` directory. It won't help the first time it happens, but it
will make the subsequent rebuilds faster (because it needs to transfer less data
from the repository). While being faster, the cache needs quite some disk space,
which might be unwanted.

Reply:
ok